src/main/java/org/kitteh/irc/client/library/Client.java

```diff
@@ -180,6 +181,16 @@
         Builder realName(@Nonnull String name);
 
         /**
+         * You can tell KICL to include a message in certain events with this option.
```

Comment:
Doesn't match javadoc approach. `Sets default interaction messages` is closer but not perfect.